build.gradle

```diff
@@ -61,6 +63,7 @@ dependencies {
 
     api 'io.ebean:ebean:12.12.0'
     api 'io.ebean:ebean-querybean:12.12.1'
+    compileOnly "net.md-5:bungeecord-api:1.17-R0.1-SNAPSHOT"
```

Comment:
Also please use the `mcVersion` variable instead of hard coding the version in the dependency.